.github/workflows/release-version.yml

```diff
@@ -0,0 +1,102 @@
+# This workflow automatically creates a release PR for the CI integration.
+
+name: Create Release PR
+
+env:
+  GIT_AUTHOR_EMAIL: 'packages@datadoghq.com'
+  GIT_AUTHOR_NAME: 'ci.datadog-ci'
+
+on:
+  workflow_dispatch:
+    inputs:
+      semver:
+        description: 'Semver argument for the version bump to do on the CI integration'
+        default: 'minor'
+        type: choice
+        options:
+          - 'minor'
```

Comment:
It's not possible to update patch or major?

Reply:
It's technically possible, but we preferred to not give the option to create a major release so easily. It would create a PR that needs to be approved though... so the approval gates us from doing something wrong.

The patch option might be useful. Let's talk about this during our daily.